component/admin/libraries/saveIcalEvent.php

```diff
@@ -215,7 +215,12 @@ public static function save($array, &$queryModel, $rrule, $dryrun = false){
 		
 		// whilst the DB field is called 'state' we use the variable 'published' in all of JEvents so must set it before the plugin
 		$vevent->published =  $vevent->state ;
-		$res = $dispatcher->trigger( 'onAfterSaveEvent' , array(&$vevent, $dryrun));
+		//$res = $dispatcher->trigger( 'onAfterSaveEvent' , array(&$vevent, $dryrun));
+
+		JPluginHelper::importPlugin("content");
+		// Trigger the onContentChangeState event. $context, $article, $isNew
+		$dispatcher->trigger('onContentAfterSave', array('com_jevents.event', $vevent, $newevent));
```

Comment:
This should not be called on a dry run! Move it until after the call to 
`if ($dryrun) return $vevent;`